src/components/Box.tsx

```diff
@@ -12,15 +12,24 @@ export interface BoxProps {
 // the number is the width of the box in terms of the number of addresses
 export default function Box(props: BoxProps): JSX.Element {
   let image;
+  let basketball;
```

Comment:
Let's just make this `const` for now. Its better practice and I don't foresee us needing different basketballs for different size boxes.